starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,41 @@
+/* ---- reset ---- */
+
+html {
+    box-sizing: border-box;
+}
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+body {
+    margin: 50px;
+    padding: 50px;
+    font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+    font-size: 18px;
+    
+}
+
+  
+div{
+    background-image: url("../images/apple-pie.jpg");
+    width:100%;
+    background-size: cover;
+    height: 400px;
+}
+h1 {
+    position: absolute;
```

Comment:
You don't need position absolute. Check here what position : absolute means: https://developer.mozilla.org/en-US/docs/Web/CSS/position